charts/node-analyzer/README.md

```diff
@@ -98,163 +98,166 @@ To check the integrity and the origin of the charts,  append the `--verify` flag
+| `nodeAnalyzer.hostScanner.resources.limits.memory`                   | Specifies the Host Scanner memory limit per node.                                                                                                                                   | `1Gi`                                                                                                                                                                 |
+| `nodeAnalyzer.hostScanner.resources.limits.ephemeral-storage`        | Specifies the Host Scanner Storage limit per node.                                                                                                                                  | `1Gi`                                                                                                                                                                 |
+| `nodeAnalyzer.hostScanner.sslVerifyCertificate`                      | Set to `false` to allow insecure connections to the Sysdig backend, such as an On-Prem deployment.                                                                                  |                                                                                                                                                                       |
+| `nodeAnalyzer.hostScanner.probesPort`                                | Specifies the port where readiness and liveness probes are exposed.                                                                                                                 | `7001`                                                                                                                                                                |
+| `nodeAnalyzer.hostScanner.scanContainers.enabled`                           | Set to `true` to scan containers                                                                                                                                                    | `false`                                                                                                                                                               |
+| `nodeAnalyzer.hostScanner.scanContainers.dockerSocketPath`          | Specifies the path to docker socket                                                                                                                                                 | ` `                                                                                                                                                                   |
+| `nodeAnalyzer.hostScanner.scanContainers.podmanSocketPath`          | Specifies the path to podman socket                                                                                                                                                 | ` `                                                                                                                                                                   |
```

Comment:
```suggestion
| `nodeAnalyzer.hostScanner.scanContainers.enabled`                           | Set to `true` to scan containers                                                                                                                                                    | `false`                                                                                                                                                               |
| `nodeAnalyzer.hostScanner.scanContainers.dockerSocketPath`          | Specifies the path to docker socket                                                                                                                                                 | ` `                                                                                                                                                                   |
| `nodeAnalyzer.hostScanner.scanContainers.podmanSocketPath`          | Specifies the path to podman socket                                                                                                                                                 | ` `                                                                                                                                                                   |
```

Indentation shall be revised. 
I'd also change the description for the paths `Specifies the optional path to docker socket. By default it will use unix://...`.